pkg/sensors/tracing/generickprobe.go

```diff
@@ -733,19 +750,6 @@ func addKprobe(funcName string, instance int, f *v1alpha1.KProbeSpec, in *addKpr
 		}
 	}
 
-	isSecurityFunc := strings.HasPrefix(funcName, "security_")
-
-	if selectors.HasOverride(f) {
-		if isSecurityFunc && in.useMulti {
-			return errFn(fmt.Errorf("error: can't override '%s' function with kprobe_multi, use --disable-kprobe-multi option",
```

Comment:
so this check is missing in `validateOverride`, I it needs `in.useMulti` so I think we need to move `validateOverride` to `addKprobe`